### PR TITLE
Added Search Handlers in the main documentation

### PR DESCRIPTION
**Pull request by zak905**
a first version of search handlers documentation - it is appended after resource documentation in the main help page of the module

### CODE REVIEW

File: omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_10/DrugSearchByMappingHandler1_10.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 
+
```
Comment (wluyima) [inline omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_10/DrugSearchByMappingHandler1_10.java]: Don't format classes you have not made changes too, can you get rid such changes please?

File: omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_10/DrugsSearchByMappingHandler1_10.java [hunk 2]
```diff
@@ -57,6 +57,7 @@
 	                + "multiple matches for the highest-priority ConceptMapType")
 	        .withRequiredParameters(REQUEST_PARAM_SOURCE)
 	        .withOptionalParameters(REQUEST_PARAM_CODE, REQUEST_PARAM_MAP_TYPES).build();
+
```
Comment (wluyima) [inline omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_10/DrugsSearchByMappingHandler1_10.java]: Same here

**Comment by wluyima** [comment]
Can you please get rid of all the format changes you made? Maven automatically formats your code using the openmrs formatter, it is hard for me to review your code

**Comment by lluismf** [comment]
I did some work on this topic in https://issues.openmrs.org/browse/RESTWS-361

**Comment by wluyima** [comment]
Thanks @lluismf , @zak905 can you take a look and see if you can incorporate Lluis' work?

**Pull request closed without merging**